### DISCUSSION TITLE
"Add single comment" is kinda starting a review

### DISCUSSION THREAD

**Discussion by evsasse** [community/community#7550] [Pull Requests] [answered]
Hey, noticed in the last couple days that when I click "Add single comment" the state of the page changes to indicate that I've started a review process, when in reality I did not.
<img width="808" alt="Screen Shot 2021-11-10 at 13 25 52" src="https://user-images.githubusercontent.com/7394653/141152592-6463999a-ff64-4318-80aa-e1dd1dc17f11.png">

The button at top changes its state like there is a pending review comment.
<img width="186" alt="Screen Shot 2021-11-10 at 13 29 21" src="https://user-images.githubusercontent.com/7394653/141153086-2a5550b1-634e-43e6-ba72-66de00dec835.png">

If I click another line to add another comment, now I only see the "Add review comment" button.
<img width="802" alt="Screen Shot 2021-11-10 at 13 28 08" src="https://user-images.githubusercontent.com/7394653/141152892-473d3cc0-561b-4af0-9437-eb25c4499594.png">

... Then I refresh the page, and everything goes back to being correct. The first comment I did was really posted, it is not pending; the button at the top doesn't any pending comments; and clicking on a line to comment does show the "Add single comment" again.

**Comment by jefflward** [comment, marked as answer]
Possible duplicate https://github.com/github/feedback/discussions/7481
